### ISSUE TITLE
Remove transformation from Sequence

### ISSUE THREAD

**Issue by vigoo** [zio/zio-schema#208]
`Sequence` could always describe a schema of `Chunk[A]` which can be converted to other collection types with `Transform`.

I made a quick attempt to do this as part of https://github.com/zio/zio-schema/pull/206 but implementing the `ZTraversal` support for the transformed sequences did not seem to be straightforward so it got reverted.

**Comment by jdegoes** [comment]
@vigoo Is this already done?

**Comment by vigoo** [comment]
No, `Sequence` still contains its own transformation: https://github.com/zio/zio-schema/blob/main/zio-schema/shared/src/main/scala/zio/schema/Schema.scala#L305-L306
